src/main/java/com/amplitude/Amplitude.java

```diff
@@ -120,10 +78,11 @@ private Object syncHttpCall(Event event) {
             OutputStream os = connection.getOutputStream();
             byte[] input = bodyString.getBytes("UTF-8");
             os.write(input, 0, input.length);
+            //os.close();
 
             int responseCode = connection.getResponseCode();
-            InputStream inputStream;
-            if (100 <= responseCode && responseCode <= 399) {
+            if (Constants.GOOD_RES_CODE_START <= responseCode &&
+                    responseCode <= Constants.GOOD_RES_CODE_END) {
```

Comment:
Why not `responseCode < 400`? You don't need 399 defined at all.

Reply:
Will do so. I don't think HTTP codes are changing any time soon